src/actions/bookmarks.js

```diff
@@ -0,0 +1,43 @@
+import { instance } from '../utils/axios';
+import handleMessages from '../utils/messages';
+import actionTypes from './types';
+
+let response;
```

Comment:
It is not appropriate to declare a variable when it is being used within a function only.

Reply:
A function should be stand-alone.

Reply:
Done